03_Uitvoering.Rmd

```diff
@@ -0,0 +1,2664 @@
+
+# Uitvoering
+
+## Keuze rasterlaag
+
+Voor de ruimtelijk gebalanceerde selectie in functie van de verspreiding van de gaHT kiezen we een GRTS-laag van Vlaanderen met level 8 ([bron](https://doi.org/10.5281/zenodo.3354405)), t.t.z. met cellen van 8192 m (figuur \@ref(fig:meetnetontwerp)).
+
+```{r meetnetontwerp, fig.cap = "Het GRTS-raster voor Vlaanderen, level 8 (8192m)", eval= TRUE, include= TRUE, echo= FALSE}
+
+#importeren van GRTS-lagen, voor het meetnet kiezen we level 8: cellen van 8192 m
+#datapath <- "./data/local/n2khab-binaire-databronnen"
+
+
+#level 4 : resolutie = 512m, level 5 :  1024m, level 6 = 2048, level 7= 4096, level 8 = 8192)
+
+raster_meetnet_poly <- read_GRTSmh_diffres(level = 8, polygon = TRUE)
+
+#oppervlakte van elk hok berekenen
+raster_meetnet_poly_opp <- raster_meetnet_poly %>% 
+  mutate(opp = as.integer(st_area(raster_meetnet_poly))) %>% 
+  st_drop_geometry() %>% 
+  dplyr::select(value, opp) %>% 
+  group_by(value) %>% 
+  summarise(totopp = sum(opp))
+
+#welke crs?
+#st_crs(habmap_polygons_gw) #lambert
+#plot(raster_meetnet_poly, main = "GRTS-raster level 8")
+#st_is_valid(raster_meetnet_poly)
+raster_meetnet_poly <- lwgeom::st_make_valid(raster_meetnet_poly)
+#st_is_valid(raster_meetnet_poly)
+
+raster_meetnet_poly_tm <- tm_shape(raster_meetnet_poly) + 
+  tm_polygons() + tm_layout(title = "GRTS-raster level 8 (8192m)" )
+
+raster_meetnet_poly_tm
+#st_crs(raster_meetnet_poly) #lambert
+
+
+#check op unieke celwaarden
+check <- raster_meetnet_poly %>% st_drop_geometry() %>% count(value) %>% filter(n > 1)
+#van bepaalde cellen zijn er dus meerdere polygonen, dit zijn rasters die door de gewestgrens verdeeld werden (bijv. streek van Baarle-Nassau) 
+
+raster_meetnet_poly <- raster_meetnet_poly %>% 
+  rename(rasterid =  value)
+
+
+```
+
+## Selectie rastercellen
+
+Op basis van deze rasterlaag maken we een overlay met de verspreiding van de vijf grondwatergroepen volgens de habitatkaart. 
+
+```{r overlay habitatkaart en GRTS-raster, eval=TRUE, echo=FALSE, warning=FALSE, include=TRUE}
+# overlay maken van de habitatkaart (enkel van GT-groepen) en het GTRS-raster (level8)
+habmap_gw_raster_overlay <- habmap_polygons_gw %>% 
+  st_intersection(raster_meetnet_poly)
+
+habmap_gw_raster_overlay <- habmap_gw_raster_overlay %>% 
+    mutate(opp = as.integer(st_area(habmap_gw_raster_overlay))) 
+
+habmap_gw_raster_overlay <- lwgeom::st_make_valid(habmap_gw_raster_overlay)
+#plot(habmap_gw_raster_overlay, main = "Voorkomen van ")
+
+habmap_gw_raster_overlay_tm <- raster_meetnet_poly_tm + 
+  tm_shape(habmap_gw_raster_overlay) + 
+  tm_fill(col = "groupnr", style = "cat", palette = "BuGn", title = "Grondwatertype") + 
```

Comment:
regarding choice of better palette, see: `tmaptools::palette_explorer()`. So an example improvement could be the `Set1` palette of brewer:

```r
  tm_fill(col = "groupnr", style = "cat", palette = "Set1", title = "Grondwatertype") + 
```

If that code doesn't work (not so familiar yet with `tmap`), `RColorBrewer::brewer.pal(5, "Set1")` should at least give you the color vector you can substitute here.